.github/CONTRIBUTING.md

```diff
@@ -17,28 +15,9 @@ Join the chat room on Gitter
 0. Make your changes.
 0. Run the tests: `npm test`.
 0. Run `git commit` and add your message describing what you have changed. If you have made changes that address an issue include the text `Closes #1` (where 1 would be the issue number) to your commit message.
-0. Submit a [pull request](https://github.com/zen-audio-player/zen-audio-player.github.io/pulls) - Travis CI will run tests to make sure nothing broke.
-0. In case the following occurs, skip to **[Syncing](#syncing)** below  
+0. Submit a [pull request](https://github.com/zen-audio-player/zen-audio-player.github.io/pulls) - Travis CI will run tests to make sure nothing broke. Once you have committed your changes locally and pushed them to your fork of the repo, a banner with a **Compare & pull request** button should appear on the main Zen Audio Player repo page to submit a pull request.
+![submit pr](/.github/submit_pr.png)
```

Comment:
Now that I took a closer look, the link for that other screenshot: `/.github/outofsync.png` has the same leading backslash and seems to link properly. Would you still recommend removing them (from both links)?